Changelog.md

```diff
@@ -1,3 +1,44 @@
+# 0.9.0.0
+
+- GHC 8.6.5 preliminary support added via the nightly build (@alanz)
+- Add `diagnosticsOnChange` config parameter, default `True`
+  (preserving prior hie behaviour). Setting it `False` only generates
+  diagnostics on file save. (#1164, @mpickering/@lorenzo)
```

Comment:
It would be nice if these PR numbers were clickable. A find-and-replace incantation should do it.